.github/workflows/main.yml

```diff
@@ -0,0 +1,66 @@
+---
+name: hs-dbus-signature CI
+
+# yamllint disable-line rule:truthy
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  # MANDATORY CHECKS THAT USE TOX
+  tox-checks:
+    strategy:
+      matrix:
+        include:
+          - task: make -f Makefile lint
+          - task: make -f Makefile test
+          - task: make -f Makefile coverage
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.7.9
+      - name: Install dependencies
+        run: pip3 install tox==3.13.2
+      - name: Run test
+        run: ${{ matrix.task }}
```

Comment:
This can be changed to `make -f Makefile ${{ matrix.task }}`. The only reason the whole command is included in the stratisd Github Actions config is that there are some variations in the actual command like adding sudo.